src/Scientist/Scientist.cs

```diff
@@ -84,7 +117,7 @@ public static T Science<T>(string name, Action<IExperiment<T>> experiment)
         /// <param name="experiment">Experiment callback used to configure the experiment</param>
         /// <returns>The value of the experiment's control function.</returns>
         public static Task<T> ScienceAsync<T>(string name, Action<IExperimentAsync<T>> experiment) =>
-            ScienceAsync(name, 1, experiment);
+             _sharedScientist.Value.ExperimentAsync(name, experiment);
```

Comment:
Looks like a stray character of whitespace was added here.

Reply:
There were a couple of these in this class, all adjusted now.